### PR TITLE
Lps 123120

### PR DESCRIPTION
**Pull request by SamZiemer**
https://issues.liferay.com/browse/LPS-123120

tests passed here: https://github.com/SamZiemer/liferay-portal/pull/771#issuecomment-722124653

Notes from @brianikim
> See issue for description. Solution is to clear the keyword parameter from the request so that the clearResultsURL actually functions as expected.
> 
> Sincerely,
> Brian

### CODE REVIEW

**Comment by drewbrokke** [comment]
ci:test:sf

**Comment by drewbrokke** [comment]
@patriciajeanperez will you please check to see if we have test coverage for this scenario? Thanks!

File: modules/apps/roles/roles-selector-web/src/main/resources/META-INF/resources/view.jsp [hunk 1]
```diff
@@ -166,11 +166,14 @@ String methodName = null;
 </liferay-util:buffer>
 
 <%
+PortletURL clearResultsURL = (PortletURL)request.getAttribute("edit_roles.jsp-portletURL");
+clearResultsURL.setParameter("keywords", StringPool.BLANK);
```
Comment (drewbrokke) [inline modules/apps/roles/roles-selector-web/src/main/resources/META-INF/resources/view.jsp:170]: Brian usually prefers a line break between a variable declaration and its usage.

@hhuijser the SF tests passed on the previous pull, is this expected? https://github.com/SamZiemer/liferay-portal/pull/771#issuecomment-722076228

**Comment by patriciajeanperez** [comment]
@drewbrokke we currently do not have clear search results in our coverage, but we will make sure to add them while we're refactoring. Thank you!

**Comment by drewbrokke** [comment]
See https://github.com/drewbrokke/liferay-portal/pull/1216

**Pull request closed without merging**